assignments/objects.js

```diff
@@ -11,54 +11,109 @@
 
 // Example format of an intern object: 1,examples@you.edu,Example,F
 const example = {
-  "id": 0,
-  "name": "Example",
-  "email": "examples@you.edu",
-  "gender": "F"
-}
+  id: 0,
+  name: "Example",
+  email: "examples@you.edu",
+  gender: "F"
+};
 
 // Write your intern objects here:
-
-
-// ==== Challenge 2: Reading Object Data ==== 
+const Mitzi = Object.create(example);
+Mitzi.id = 1;
+Mitzi.name = "Mitzi";
+Mitzi.email = "mmelloy0@psu.edu";
+Mitzi.gender = "F";
+
+const Kennan = Object.create(example);
+Kennan.id = 2;
+Kennan.name = "Kennan";
+Kennan.email = "kdiben1@tinypic.com";
+Kennan.gender = "M";
+
+const Keven = Object.create(example);
+Keven.id = 2;
+Keven.name = "Keven";
+Keven.email = "kmummery2@wikimedia.org";
+Keven.gender = "M";
+
+const Gannie = Object.create(example);
+Gannie.id = 3;
+Gannie.name = "Gannie";
+Gannie.email = "gmartinson3@illinois.edu";
+Gannie.gender = "M";
+
+const Antonietta = Object.create(example);
+Antonietta.id = 4;
+Antonietta.name = "Antonietta";
+Antonietta.email = "adaine5@samsung.com";
+Antonietta.gender = "F";
+
+// ==== Challenge 2: Reading Object Data ====
 // Once your objects are created, log out the following requests from HR into the console:
```

Comment:
even though your objects are throwing errors, I know your code below would otherwise work so I'll count this as good and done.